src/System.Data.SqlClient/src/System.Data.SqlClient.csproj

```diff
@@ -20,4 +13,18 @@
       <GeneratePlatformNotSupportedAssemblyMessage Condition="'$(TargetPlatformIdentifier)' != 'windows'">SR.PlatformNotSupported_System_Data_SqlClient</GeneratePlatformNotSupportedAssemblyMessage>
     </PropertyGroup>
 
+    <!-- Floating package information -->
+    <PropertyGroup>
+      <IsPackable>false</IsPackable>
+      <VersionPrefix>4.8.6</VersionPrefix>
+      <VersionPrefix Condition="'$(IsPackable)' == 'true'">4.9.0</VersionPrefix>
+      <AssemblyVersion Condition="$([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETFramework'">4.6.1.6</AssemblyVersion>
+      <AssemblyVersion Condition="'$(IsPackable)' == 'true' and $([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETFramework'">4.6.2.0</AssemblyVersion>
+      <PackageValidationBaselineVersion>4.8.6</PackageValidationBaselineVersion>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <PackageReference Include="Microsoft.DotNet.Build.Tasks.TargetFramework" PrivateAssets="all" />

```

Comment:
I see. You're removing it from the global targets file and placing it only in the csprojs that use it.

Can you remind me what's used from this package reference?

Reply:
This package is necessary to be able to target RIDs in TFMs, i.e. `net9.0-unix` which aren't officially supported by the .NET SDK.